tests/Doctrine/Tests/Common/Annotations/DocParserTest.php

```diff
@@ -1300,6 +1300,32 @@ public function testSupportsEscapedQuotedValues()
         $this->assertTrue($result[0] instanceof Name);
         $this->assertEquals('"bar"', $result[0]->foo);
     }
+
+    /**
+     * @see http://php.net/manual/en/mbstring.configuration.php
+     * mbstring.func_overload can be changed only in php.ini
+     * so for testing this case instead of skipping it you need to manually configure your php installation
+     */
+    public function testMultiByteAnnotation()
+    {
+        $overloadStringFunctions = 2;
```

Comment:
No, there is two different links, but you can go from page in code to second page :-)